tests/integration/valkey-cli.tcl

```diff
@@ -833,6 +843,37 @@ if {!$::tls} { ;# fake_redis_node doesn't support TLS
         close_cli $fd
     }
 
+    test "valkey-cli make sure selected db survives connection drops" {    
+        set fd [open_cli]
+        
+        # Target database
+        set db 9
+
+        # Select the database        
+        assert_equal "OK" [run_command $fd "select $db"]
```

Comment:
Fails in cluster mode?

Reply:
`cluster:skip` tag is set. but now it won't be needed as $::dbnum will be used (`select 0` should work in cluster mode).